lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -17,11 +17,32 @@ export default function LMSGrader({
   children,
   assignmentName,
   courseName,
-  students,
+  students: unorderedStudents,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  // A sorted list of students. Students are sorted by
```

Comment:
This comment could probably be made more concise: "Students sorted by `displayName`" or similar.

Reply:
agreed